android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java

```diff
@@ -382,5 +382,15 @@ public void setData(Chart chart, ReadableMap propMap) {
         chart.invalidate();
     }
 
-
+    /**
+     * details: https://github.com/PhilJay/MPAndroidChart/wiki/Highlighting
+     */
+    @ReactProp(name = "highlightValue")
+    public void setHighlightValue(Chart chart, ReadableMap propMap) {
+        chart.highlightValue(
+            (float) propMap.getDouble("x"),
+            (int) propMap.getInt("dataSetIndex"),
+            (boolean) propMap.getBoolean("callListener")
```

Comment:
out of curiosity, why do these need to be cast if the function name is specific to the type?